File: PYTHON/Gpu.py
class Gpu:
    def __init__(self, memory_gb=0, tipe_gpu="", merk="", nama=""):
        self.memory_gb = memory_gb
        self.tipe_gpu = tipe_gpu
        self.merk = merk
        self.nama = nama

    def set_memory_gb(self, memory_gb):
        self.memory_gb = memory_gb

    def set_tipe_gpu(self, tipe_gpu):
        self.tipe_gpu = tipe_gpu

    def get_memory_gb(self):
        return self.memory_gb

    def get_tipe_gpu(self):
        return self.tipe_gpu

    def get_merk(self):
        return self.merk

    def get_nama(self):
        return self.nama

File: CPP/Gpu.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class Gpu : public Komponen
{
private:
    int memoryGB;
    string tipeGpu;
public:
    Gpu() {}

    Gpu(int memoryGB, string tipeGpu, string merk, string nama) : Komponen(merk, nama)
    {
        this->memoryGB = memoryGB;
        this->tipeGpu = tipeGpu;
    }

    void setMemoryGB(int memoryGB)
    {
        this->memoryGB = memoryGB;
    }

    void setTipeGpu(string tipeGpu)
    {
        this->tipeGpu = tipeGpu;
    }

    int getMemoryGB()
    {
        return this->memoryGB;
    }

    string getTipeGpu()
    {
        return this->tipeGpu;
    }

    ~Gpu() {}
};
